packages/examples/fortune/recording-oracle/src/plugins/curses.ts

```diff
@@ -1,465 +1,14 @@
 import fp from 'fastify-plugin';
 import { FastifyPluginAsync } from 'fastify';
+import { pipeline } from '@xenova/transformers';
 
-const curses = [
```

Comment:
I think we agreed on having a two-stage approach, where both a wordlist and the model prediction is checked. You mentioned an extended / improved list of cursewords.

Reply:
Alright, misunderstanding then.